doc/source/examples/dem/rotating-drum/rotating-drum.rst

```diff
@@ -33,7 +32,7 @@ Parameter File
 Mesh
 ~~~~~
 
-In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius and half-length, respectively. Therefore, the specified grid arguments create a cylinder with a diameter of 0.24 m and a length of 0.36 m. The grid is refined 4 times to reach the desired cell size to particle diameter ratio (see packing in ball example for more details). The ``expand particle-wall contact search`` is used in concave geometries to enable extended particle-wall contact search with boundary faces of neighbor cells for particles located in each boundary cell. Between two consecutive contact search steps, where the containing cells of the particles are updated (i.e. particles are mapped into cells), particles located close to the cell boundaries may change cells. If this situation occurs for a particle on a boundary, the particle-wall collision is calculated using the information (normal vector and location of the vertices) of the previous boundary cell. Hence, when the containing cell of the particle updates, the particle may already have a large overlap with the new cell that leads to a large contact force/velocity of the particles. We use ``expand particle-wall contact search`` to avoid this undesired situation.
+In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius and half-length, respectively. Therefore, the specified grid arguments create a cylinder with a diameter of 0.24 m and a length of 0.36 m. The grid is refined 4 times to reach the desired cell size to particle diameter ratio (see packing in ball example for more details). The ``expand particle-wall contact search`` is used in concave geometries to enable extended particle-wall contact search with boundary faces of neighbor cells for particles located in each boundary cell. 
 
 .. code-block:: text
```

Comment:
The `subsection mesh` in the example .prm file is:

subsection mesh
  set type                                = dealii
  set grid type                           = subdivided_cylinder
  set grid arguments                      = 4: 0.12:0.18
  set initial refinement                  = 4
  set expand particle-wall contact search = true
end

Reply:
The young modulus is 1e7 instead of 1e8 (walls and particles)

Reply:
The insertion info is outdated in the `rotating-drum.prm` file (I know it doesn't play a role since `set inserted number of particles at each time step = 0`, but still)

Reply:
Load balancing parameters do not match on both prm files. That is fine, but it is not clear in the example what is in the load and what is in the rotating drum file. Not a big deal, but I would maybe point that info out through the example.

Reply:
The `subsection simulation control` in the .prm file of the example is:

subsection simulation control
  set time step         = 1e-5
  set time end          = 12
  set log frequency     = 1000
  set output frequency  = 1000
  set output boundaries = false
  set output path       = ./output/
end